syntax = "proto3";

package aiscopre.user;

service UserService {
  rpc Register(RegisterRequest) returns (RegisterResponse);
  rpc Login(LoginRequest) returns (LoginResponse);
}

message RegisterRequest {
  string username = 1;
  string password = 2;
}

message RegisterResponse {
  string user_id = 1;
}

message LoginRequest {
  string username = 1;
  string password = 2;
}

message LoginResponse {
  string user_id = 1;
  string token = 2;
}

